_posts/2016-10-21-sweet-break-before-break-102.md

```diff
@@ -0,0 +1,120 @@
+---
+
+title: 'A sweet break before the break: TGIF (102)'
+author: Robert
+layout: blog-post-tgif
+comments: false
+
+---
+
+Thank Grohl, Gauss, Glob, Galileo and Science it’s Friday! This Friday’s little bit of knowledge, fascination, and fun is curated for you by [Rob](https://twitter.com/gibbiv) :)
+
+## Editors’ Note: A short hiatus
+
+Last week I hopped on a call with [Cecily](https://twitter.com/skeskali), [Gregor](https://twitter.com/gr2m), and [Jenn](https://twitter.com/jennwrites) to discuss the future of TGIF, Hoodie’s lazy Friday afternoon read started by [Lena](https://twitter.com/lrnrd) in [August 2013](http://hood.ie/blog/tgif-hoodies-top-links-of-the-week-1.html). After an hour-long chat with these amazing peeps we decided to take a step back and rethink how Hoodie’s weekly newsletter is created, delivered and promoted.
+
+Some of the things we discussed were TGIF having it’s own platform (separate from the blog), opening each issue up to multiple contributors (one contributor per section), making TGIF more accessible (with an email subscription option), and reaching out to publishers of content we share (to spread awareness).
+
+I’m grateful to be part of this project and look forward to making TGIF one of the most anticipated newsletters in open source, intellect and fun! Now for what you may have missed over the past 2 weeks...
+
+## Open Source
+
+[Obama’s Facebook Messenger bot is open source](https://www.whitehouse.gov/blog/2016/10/13/removing-barriers-constituent-conversations) (whitehouse.gov)
+
+Obama reads ten letters a day, but he can’t read em’ all, so he decided to scale responses with the Obama bot: a handy Drupal [module](https://github.com/WhiteHouse/fb_messenger_bot) that you (or any government) can now use to build a chat bot on Facebook's Messenger platform.
+
+[FreeBSD stable/11 branch released](https://www.freebsd.org/releases/11.0R/announce.html) (freebsd.org)
```

Comment:
Let's always spell out abbreviations on the first reference. This saves people a Google search (or, for me, a DuckDuckGo search) to learn something new. BSD, I just learned right now, stands for `Berkeley Software Distribution`. So something like: `FreeBSD (Berkeley Software Distribution) stable/11 branch released`